app/util/AWS.scala

```diff
@@ -87,6 +100,27 @@ class AWSConfig @Inject() (config: Configuration) {
     null
   )
 
+  private def createUploadSTSClient() = {
+    val provider = stage match {
+      case "DEV" =>
```

Comment:
Watch this space, AWS have promised they'll be fixing the problems that prevent Janus from issuing credentials that can call assumeRole. It'll definitely be worth changing this when that's possible.